homeassistant/components/device_tracker/__init__.py

```diff
@@ -211,13 +220,15 @@ class DeviceTracker(object):
     """Representation of a device tracker."""
 
     def __init__(self, hass: HomeAssistantType, consider_home: timedelta,
-                 track_new: bool, devices: Sequence) -> None:
+                 track_new: bool, defaults: dict,
+                 devices: Sequence) -> None:
         """Initialize a device tracker."""
         self.hass = hass
         self.devices = {dev.dev_id: dev for dev in devices}
         self.mac_to_dev = {dev.mac: dev for dev in devices if dev.mac}
         self.consider_home = consider_home
-        self.track_new = track_new
+        self.track_new = defaults.get(CONF_TRACK_NEW, track_new)
+        self.hide_if_away = defaults.get(CONF_AWAY_HIDE, DEFAULT_AWAY_HIDE)
```

Comment:
We shouldn't store it here. We should just store the defaults object and use it inside `async_see`.